R/checks.R

```diff
@@ -286,7 +286,25 @@ check_workflow <- function(x, pset = NULL, check_dials = FALSE) {
     }
   }
 
-  mod <- extract_spec_parsnip(x)
+  mod <- hardhat::extract_spec_parsnip(x)
+
+  to_be_tuned <- hardhat::extract_parameter_set_dials(mod)
```

Comment:
Note that this is a separate parameter set from the one used earlier in the function. Is not sensitive to the user's `param_info` input and called only with respect to the model spec, rather than the workflow.